common/menu.c

```diff
@@ -29,6 +30,39 @@ void launchMenuEntryTask(menuEntry_s* arg) {
         launchMenuEntry(me);
 }
 
+void toggleStarState(menuEntry_s* arg) {
+    menuEntry_s* me = arg;
+    if (me->starred)
+    {
+        if (fileExists(me->starpath))
+            remove(me->starpath);
+    } else {
+        if (!fileExists(me->starpath))
+        {
+            int fd = open(me->starpath,O_CREAT|O_WRONLY, ACCESSPERMS);
```

Comment:
Was it really necessary to bring up POSIX I/O functions? Could this just be `fopen(me->starpath, "w")` or am I mistaken?

Reply:
Honestly I copy/pasted from somewhere in netloader.c - now changed to fopen, works fine.